Third Part of Diplom Project

kosalexkos opened the pull request:
Implemented the feature to comment the events for the diploma project

kosalexkos committed:
Third Part - Initial Commit

Implemented the comment to the events feature for the diploma project

kosalexkos committed:
Update README.md

added link to the PR

felixoff commented on ewm-service/src/main/java/ru/practicum/comments/dto/NewCommentDto.java:
```diff
@@ -0,0 +1,21 @@
+@NoArgsConstructor
+@AllArgsConstructor
+public class NewCommentDto {
+    @NotNull
```
Зачем аннотация NotNull если уже есть NotBlank? Почитай про них

felixoff commented on ewm-service/src/main/java/ru/practicum/comments/model/Comment.java:
```diff
@@ -0,0 +1,34 @@
+    private User user;
+    @ManyToOne
+    @JoinColumn(name = "event_id")
+    private Event event;
```
Вместо множественных private лучше использовать @FieldDefaults

kosalexkos committed:
After review

Got rid of redundant annotation and used more efficient way of setting access modifiers for entities fields
